docs/TESTING.md

```diff
@@ -0,0 +1,25 @@
+# Testing
+
+### E2E tests on BrowserStack
+
+#### Prerequisites
+
+-   [BrowserStack](https://www.browserstack.com) account
+
+### Running tests
+
+This will run all the tests in `showcase/src/e2e-test`
```

Comment:
Could  you create the same folder in the Starter and move all the E2E tests there? So our 2 apps are consistent ;)

Reply:
If I'm going to implement this in the starter, I should probably provide some more documentation. I think it is best if I also provide a solution to run e2e-tests without a browserstack configuration.